lib/src/ui/movies_page.dart

```diff
@@ -14,11 +18,225 @@ class _MoviesPageState extends State<MoviesPage> {
+                      filled: true,
+                      fillColor: Colors.white12),
+                ),
+              ),
+            ],
+          ),
+        ),
+      ),
+      body: SingleChildScrollView(
+        child: Padding(
+          padding: const EdgeInsets.all(UiConstants.bodyPadding),
+          child: Column(
+            children: <Widget>[
+              Align(
+                alignment: Alignment.centerLeft,
+                child: Padding(
+                  padding: const EdgeInsets.only(
+                      bottom: UiConstants.movieTypeTextPadding),
+                  child: new Text(
+                    'Trending Movies',
+                    style: TextStyle(
+                        fontSize: UiConstants.movieTypeTitleFontSize,
+                        color: Colors.grey,
+                        fontFamily: 'Play-Bold'),
+                  ),
+                ),
+              ),
+              StreamBuilder(
+                stream: _moviesBloc.swiper,
+                builder: (
+                  context,
+                  AsyncSnapshot<Movies> snapshot,
+                ) {
+                  return snapshot.hasData
+                      ? _swiper(
+                          snapshot,
+                        )
+                      : Center(
+                          child: Padding(
+                            padding: const EdgeInsets.only(
+                                top: UiConstants
+                                    .swiperProgressIndicatorTopPadding),
+                            child: CircularProgressIndicator(
+                              strokeWidth:
+                                  UiConstants.progressIndicatorStrokeWidth,
+                              backgroundColor: Colors.purple,
+                              valueColor: new AlwaysStoppedAnimation<Color>(
+                                  Colors.black),
+                            ),
+                          ),
+                        );
+                },
+              ),
+              StreamBuilder(
+                stream: _moviesBloc.stream,
+                builder: (
+                  context,
+                  AsyncSnapshot<Movies> snapshot,
+                ) {
+                  return snapshot.hasData
+                      ? buildList(
+                          snapshot,
+                        )
+                      : Padding(
+                          padding: const EdgeInsets.only(
+                              top: UiConstants.gridProgressIndicatorTopPadding),
+                          child: Center(
+                            child: CircularProgressIndicator(
+                              strokeWidth:
+                                  UiConstants.progressIndicatorStrokeWidth,
+                              backgroundColor: Colors.purple,
+                              valueColor: new AlwaysStoppedAnimation<Color>(
+                                  Colors.black),
```

Comment:
the "new" identifier is not necessary, and missed a trailing comma on the second line

Reply:
removed and added